CHANGELOG.md

```diff
@@ -7,6 +7,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Added
+- Support for computed members
```

Comment:
```suggestion
- Add support for computed members
```

Reply:
I appreciate you would like to include the word `Add` but I intentionally didn't include it since the heading of the section is `Added`. So, if you read the heading + bullet point, it reads: Added support for computed members :grin:

Reply:
this is @mariux's convention, check any changelog in the past half year.
and also, i would agree and accept your stance if you would do this with all the other [sections](https://github.com/mineiros-io/terraform-google-cloud-run-iam/blob/71ed7c2bea762dcb248b89095197f388345928b8/CHANGELOG.md?plain=1#L14) as well :)

Reply:
hahaha, nice catch. I failed at being consistent.